data_structures/read_group.wdl

```diff
@@ -0,0 +1,386 @@
+version 1.1
+
+# See the `read_groups` `parameter_meta` for definitions of each field
```

Comment:
I think the `read_group` field definitions need to live within this file. Probably as header comments. They are currently defined in `rnaseq-standard-fastq`, which should instead point to this file "for more information".

Reply:
I've brought them over as a header comment and also documented inline in the `struct` definition. That might be overkill though.

Reply:
I gather it's the exact same information being documented in both places? That sounds like a mistake of only updating one is bound to happen in the future. I think just the header comments. Maybe a comment above the struct definition saying "see header"?

Reply:
My assumption was that the read group fields are unlikely to update in the future.

Reply:
That is true. But what do we gain from these inline comments? Who are they meant for? I figure anyone who might see these inline comments would have to at least skimmed past the header comments. Is there a case I'm not considering?